op-node/p2p/gossip.go

```diff
@@ -82,15 +82,20 @@ func blocksTopicV4(cfg *rollup.Config) string {
 	return fmt.Sprintf("/optimism/%s/3/blocks", cfg.L2ChainID.String())
 }
 
-// BuildSubscriptionFilter builds a simple subscription filter,
-// to help protect against peers spamming useless subscriptions.
-func BuildSubscriptionFilter(cfg *rollup.Config) pubsub.SubscriptionFilter {
-	return pubsub.NewAllowlistSubscriptionFilter(
+func allBlocksTopics(cfg *rollup.Config) []string {
```

Comment:
Out of curiosity, is there an approach for when we expose functions? (I've been out of monorepo goland for a while now.)

Reply:
What I mean is when we make functions visible by using pascal as opposed to camel casing?

Reply:
I usually default to not exporting the function, to avoid bloating the API of the package and making breaking changes more likely. OTOH we could expose this if we thought it might be useful. I don't think this code is used as library code much though, so I'm not sure what the use case is right now.